### DISCUSSION TITLE
2.6.1版本下载nuget包后发现包中包含pdb文件，单步调试时提示未找到源，倒退到2.5.2版本单步调试没有问题

### DISCUSSION THREAD

**Discussion by jinghuian** [dotnetcore/FreeSql#906] [General]
![微信图片_20211014093108](https://user-images.githubusercontent.com/26103751/137235325-bc3053d1-6896-40a2-91c3-b0cfdf9bb8c2.png)
2.6.1版本下载nuget包后发现包中包含pdb文件，单步调试时提示未找到到源，倒退到2.5.2版本单步调试没有问题
**在不联网状态提示未找到源，联网状态下在C:\Users\usersname\AppData\Local\SourceServer\文件夹下下载到源码文件，
单步进入，目前不理解为什么**
FreeSql.2.6.100
FreeSql.DbContext.2.6.100
FreeSql.Extensions.LazyLoading.2.6.100
FreeSql.Provider.SqlServer.2.6.100
FreeSql.Repository.2.6.100
以上都包含pdb文件
提示如下：
为“C:\Users\28810\Desktop\github\FreeSql\FreeSql\Internal\CommonProvider\BaseDbProvider.cs”查找源。Checksum: SHA256 {c3 53 af a8 f9 bb c1 eb 4f 28 3 9a dc 10 57 ea ed 9b f8 81 f3 d2 ba 4f c3 66 91 df 1c 78 9f 8}
文件“C:\Users\28810\Desktop\github\FreeSql\FreeSql\Internal\CommonProvider\BaseDbProvider.cs”不存在。
正在脚本文档中查找“C:\Users\28810\Desktop\github\FreeSql\FreeSql\Internal\CommonProvider\BaseDbProvider.cs”...
正在项目中查找“C:\Users\28810\Desktop\github\FreeSql\FreeSql\Internal\CommonProvider\BaseDbProvider.cs”。
在项目中未找到该文件。
搜索符号文件中嵌入的文档。
找不到嵌入文档。
活动解决方案的调试源文件设置指明调试器不会要求用户查找文件: C:\Users\28810\Desktop\github\FreeSql\FreeSql\Internal\CommonProvider\BaseDbProvider.cs。
调试器未能找到源文件“C:\Users\28810\Desktop\github\FreeSql\FreeSql\Internal\CommonProvider\BaseDbProvider.cs”。

**Comment by densen2014** [comment]
关掉vs自动调试下载源码功能即可

**Reply by jinghuian** [reply]
![微信图片_20211014110446](https://user-images.githubusercontent.com/26103751/137243953-da13ef6a-1323-4d74-839b-f88d852897c8.png)
是这样吗？不起作用呀？

**Reply by jinghuian** [reply]
![微信截图_20211014110919](https://user-images.githubusercontent.com/26103751/137244452-9fcbdca9-629b-4c4b-bd2a-86957bd11002.png)

**Reply by jinghuian** [reply]
不知道在哪里配置才能禁用该功能

**Comment by luoyunchong** [comment]
相比上一个版本，这个多配置了[ https://github.com/dotnetcore/FreeSql/blob/master/Directory.Build.props]( https://github.com/dotnetcore/FreeSql/blob/master/Directory.Build.props) 。但2.6.100并没有配置 `IncludeSymbols`,`SymbolPackageFormat` 后来才增加。。该功能应该是完成 符号调试 [https://github.com/dotnetcore/FreeSql/issues/679](https://github.com/dotnetcore/FreeSql/issues/679)

```
<IncludeSymbols>true</IncludeSymbols>
<SymbolPackageFormat>snupkg</SymbolPackageFormat>
```

**Reply by jinghuian** [reply]
不知道在哪里配置才能禁用该功能
